src/styles/blocks/main-page.scss

```diff
@@ -0,0 +1,11 @@
+.main-page {
+  &__container {
+    max-width: $main-page-container-width;
+    min-width: $main-page-container-width;
+    margin: 0 auto;
+    padding: 50px 0;
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+  }
+}
```

Comment:
Let's use one block here and create BEM element for `main-page` block to set margins there. It looks strange that we are setting outer indents between element and parent container